cairo/ethereum/cancun/vm/instructions/system.cairo

```diff
@@ -125,3 +164,192 @@ func return_{range_check_ptr, evm: Evm}() -> EthereumException* {
     let ok = cast(0, EthereumException*);
     return ok;
 }
+
+func generic_create{
+    process_create_message_label: felt*,
+    process_message_label: felt*,
```

Comment:
why is this needed?

Reply:
to circumvent circular dependencies issues

Reply:
it's not showing in the GH CI but I meant, do you need the process_message_label too?

Reply:
ah ok, no